cmd/opdoc/opdoc.go

```diff
@@ -248,6 +248,7 @@ type OpRecord struct {
 	Doc           string
 	DocExtra      string `json:",omitempty"`
 	ImmediateNote string `json:",omitempty"`
+	Version       int
```

Comment:
@barnjamin Optional - The _suggestion_ aligns data type with `OpSpec.Version`.   I don't feel strongly though I think it may remove questions to keep the same data type.

```suggestion
	Version       uint64
```

Reply:
did this separately to fix the type cast below as well